joomla/joomla-cms
# Email address: authorise single quote in local-part (long standing bug)

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/12804

### Summary of Changes

Authorise the use of a single quote in the local-part of an email address by changing `’` to `'` in the regex.
Single quote is authorised by RFC-2822 while `#8217` `’` is not

### Testing Instructions
Create or change the email of a user to include a single quote, for example
`John.O'Connor@foobar.com`
It will now validate and save ok.

Display that type of mail in frontend in an article.
Make sure the mailcloak plugin is enabled.

Check source: the address is cloaked OK
